src/main/java/org/killbill/billing/client/model/SubscriptionUsage.java

```diff
@@ -0,0 +1,76 @@
+package org.killbill.billing.client.model;
+
+import java.util.List;
+import java.util.UUID;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class SubscriptionUsage  {
```

Comment:
Let's follow the current naming convention (i.e. the Kill Bill model name minus the Json suffix). You can update the same PR, that's fine.